contracts/test/mocks/MockCustomEntitlement.sol

```diff
@@ -9,12 +9,19 @@ contract MockCustomEntitlement is ICustomEntitlement {
   constructor() {}
 
   function setEntitled(address[] memory user, bool userIsEntitled) external {
```

Comment:
this was a bug yeah? Can you please change the parameter name to `users`?

Reply:
ok fixed!